software/enclosed.yml

```
@@ -0,0 +1,13 @@
+name: "Enclosed"
+website_url: "https://enclosed.cc/"
+source_code_url: "https://github.com/CorentinTh/enclosed"
+description: "A minimalistic web application designed for sending private and secure notes."
+licenses:
+  - Apache-2.0
+platforms:
+  - Nodejs
+  - Docker
```

Small change to reflect docs

```suggestion
platforms:
  - Docker
  - Nodejs
```